From ed2cae2871470aadba9565fc27eff0cbfebbbdb6 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:10:58 -0400
Subject: [usb] pl2303: make type data const

Message-id: <1410887463-20674-587-git-send-email-dzickus@redhat.com>
Patchwork-id: 95631
O-Subject: [RHEL7 PATCH 586/591] USB: pl2303: make type data const
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit b693468155f5650dcaf01089ecf2b56ab66f9271
Author: Johan Hovold <jhovold@gmail.com>
Date:   Thu Jan 2 22:49:20 2014 +0100

    USB: pl2303: make type data const

    Declare constant device-type data as const.

    Signed-off-by: Johan Hovold <jhovold@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/pl2303.c b/drivers/usb/serial/pl2303.c
index 6ec9f3c..0a01fb6 100644
--- a/drivers/usb/serial/pl2303.c
+++ b/drivers/usb/serial/pl2303.c
@@ -143,7 +143,7 @@ struct pl2303_type_data {
 };
 
 struct pl2303_serial_private {
- struct pl2303_type_data *type;
+ const struct pl2303_type_data *type;
  unsigned long quirks;
 };
 
@@ -155,7 +155,7 @@ struct pl2303_private {
  u8 line_settings[7];
 };
 
-static struct pl2303_type_data pl2303_type_data[TYPE_COUNT] = {
+static const struct pl2303_type_data pl2303_type_data[TYPE_COUNT] = {
  [TYPE_01] = {
   .max_baud_rate = 1228800,
   .quirks =  PL2303_QUIRK_LEGACY,
-- 
1.7.1